stories/radio/radio.stories.js

```diff
@@ -25,45 +25,57 @@ export const primary = () => `<div style="display:flex;justify-content:space-bet
         <legend class="fd-fieldset__legend">Radio Buttons Cozy Mode</legend>
         <div class="fd-form__group">
             <div class="fd-form-item">
-                <input type="radio" class="fd-radio" id="pDidh761" name="radio1" checked>
-                <label class="fd-radio__label" for="pDidh761">
-                    Field label
-                </label>
+                <div class="fd-radio__wrapper">
+                    <input type="radio" class="fd-radio" id="pDidh761" name="radio1" checked>
```

Comment:
I am not sure we are following BEM with this approach. In this case you have `fd-radio__wrapper` as a parent element for `fd-radio` which is a BEM violation. A class with `__something` must be an element of a block class